nemo/core/neural_types/elements.py

```diff
@@ -192,4 +198,34 @@ class CategoricalValuesType(PredictionsType):
 
 
 class MaskType(PredictionsType):
-    """Element type to represent boolean mask"""
+    """Element type to represent a boolean mask"""
+
+
+class TargetType(ElementType):
+    """
+        Element type to represent a target value.
+    """
+
+
+class ClassificationTargetType(ElementType):
```

Comment:
bug ;)

Reply:
What's the difference between TargetType and ClassificationTargetType?